app/views/pictures/edit.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_for @picture do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <%= f.submit %>
+<% end %>
```

Comment:
This form could also be in a partial: that way you could render the same partial form for both `edit` and `new`